po/zh_CN.po

```diff
@@ -132,153 +131,154 @@ msgstr "更改家区域密码需要认证。"
 msgid ""
 "Home of user %s is currently absent, please plug in the necessary storage "
 "device or backing file system."
-msgstr ""
+msgstr "用户 %s 的家目录当前不存在，请插入必要的存储设备或下层文件系统。"
 
 #: src/home/pam_systemd_home.c:292
 #, c-format
 msgid "Too frequent login attempts for user %s, try again later."
-msgstr ""
+msgstr "用户 %s 的登录尝试过于频繁，请稍后重试。"
 
 #: src/home/pam_systemd_home.c:304
 msgid "Password: "
-msgstr ""
+msgstr "密码： "
 
 #: src/home/pam_systemd_home.c:306
 #, c-format
 msgid "Password incorrect or not sufficient for authentication of user %s."
-msgstr ""
+msgstr "用户 %s 的密码不正确或不足以完成认证。"
 
 #: src/home/pam_systemd_home.c:307
 msgid "Sorry, try again: "
-msgstr ""
+msgstr "抱歉，请重试： "
 
 #: src/home/pam_systemd_home.c:329
 msgid "Recovery key: "
-msgstr ""
+msgstr "恢复密钥： "
 
 #: src/home/pam_systemd_home.c:331
 #, c-format
 msgid ""
 "Password/recovery key incorrect or not sufficient for authentication of user "
 "%s."
-msgstr ""
+msgstr "用户 %s 的密码/恢复密钥不正确或不足以完成认证。"
 
 #: src/home/pam_systemd_home.c:332
 msgid "Sorry, reenter recovery key: "
-msgstr ""
+msgstr "抱歉，请重新输入恢复密钥： "
 
 #: src/home/pam_systemd_home.c:352
 #, c-format
 msgid "Security token of user %s not inserted."
-msgstr ""
+msgstr "用户 %s 的安全令牌未插入。"
 
 #: src/home/pam_systemd_home.c:353 src/home/pam_systemd_home.c:356
 msgid "Try again with password: "
-msgstr ""
+msgstr "请尝试重新输入密码： "
```

Comment:
Google translate says that this means "Please try re-entering your password", which is related but actually a different prompt.

Reply:
This string is kind of vague when it comes with the context that its near the string
`Password incorrect or not sufficient, and configured security token of user %s not inserted.`
I previously thought it could expressed that asking user to re-entering the password. But after checking the source code, I think this part of code express that when the security token is not precense, asking user to authenticate with password.